lab-dan/lib/greet.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {};
```

Comment:
Also well done. You could accomplish the if/else logic with a ternary operator as well. :-)